app/models/user.rb

```diff
@@ -1,6 +1,6 @@
 class User < ApplicationRecord
   has_many :ratings
-  validates :email, uniqueness: true
+  validates :email, presence: true, uniqueness: true
```

Comment:
email value should not be blank before we attempt to lowercase it.
presence: true will make sure lowercase emails that are non blank.